examples/redis-rest/k8s-resources/apirule.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: gateway.kyma-project.io/v2alpha1
```

Comment:
I would suggest migrating APIRule to version 2
https://kyma-project.io/#/api-gateway/user/custom-resources/apirule/04-10-apirule-custom-resource

Reply:
It will not work for users on Regular channel. It should be available from the 5th of May.